### PR TITLE
Propose translation to Spanish

### PR DESCRIPTION
**Pull request by danirod**
Hi. I added a translation to Spanish for this essay.

I used the French translation as a starting point rather than the English version.

### CODE REVIEW

File: es/index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        ¿hay alguien que sepa de Java?
+      </p>
+    </blockquote>
+    <p>
+      Esto está mal por muchas razones. 
+      o que esta persona está <em>en realidad</em> preguntando es,
```
Comment (Lonami) [inline es/index.html:98]:
```suggestion
      Lo que esta persona está <em>en realidad</em> preguntando es,
```

File: es/index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        ¿hay alguien que sepa de Java que quiera ceder a mirar mi problema,
+        sea lo que sea,
+        incluso si al final resulta que no tiene que ver con Java,
+        o incluso si alguien que no sepa de Java supiese la respuesta a mi pregunta?
+      </p>
+    </blockquote>
+    <p>
+      Hay un montón de razones por las que alguien que SÍ sepa la respuesta ya no lo va a hacer.
```
Comment (Lonami) [inline es/index.html:110]:
```suggestion
      Hay un montón de razones por las que alguien que SÍ sepa la respuesta ya no va a responder.
```

File: es/index.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+      </p>
+    </blockquote>
+    <p>
+      ...lo cual es vago.
+      Si no tienes interés en hacer algo de trabajo para solucionar tu problema,
+      ¿por qué deberían otros? 
+    </p>
+    <p>
+      La solución es no pedir permiso para preguntar,
+      sino preguntar.
```
Comment (Lonami) [inline es/index.html:137]:
```suggestion
      sino preguntar directamente.
```

**Comment by danirod** [comment]
Thanks for the feedback @Lonami! :)

**Pull request merged**